SmsVerification/app/src/main/res/layout/activity_phone_number.xml

```diff
@@ -9,7 +9,7 @@
     android:paddingLeft="@dimen/activity_horizontal_margin"
     android:paddingRight="@dimen/activity_horizontal_margin"
     android:paddingTop="@dimen/activity_vertical_margin"
-    tools:context="com.google.samples.smartlock.sms_verify.PhoneNumberActivity"
+    tools:context="com.google.samples.smartlock.sms_verify.PhoneNumberVerifier"
```

Comment:
Is it intentional that the value for tools:context is changed?

Reply:
could help double check whether this is necessary? thanks

Reply:
Checked that PhoneNumberActivity didn't exist before your pull request so the change looks good to me. Thanks.